includes/apple-exporter/components/class-table.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * Contains a component class representing a table.
+ *
+ * @package Apple_News
+ * @since 1.4.0
+ */
+
+namespace Apple_Exporter\Components;
+
+/**
+ * A component class representing a table.
+ *
+ * @since 1.4.0
+ */
+class Table extends Component {
+
+	/**
```

Comment:
are the lines within the class definition padded with spaces? It's hard to tell in GH sometimes what is a space and what is a tab.

Reply:
I just checked, and it's tabs all the way down.

🌍 
🐢 
🐢 
🐢